lib/framework.js

```diff
@@ -109,6 +109,10 @@ class Framework {
 			test: "test"
 		};
 
+		if (!!this.config.client.ui5.useIframe === false) {
```

Comment:
This doesn't allow setting `useIframe` to `false`, right? It will always be set to `true`.

I would rather explicitly check for "false" and otherwise use "true" (default)
```js
if (this.config.client.ui5.useIframe !== false) {
  this.config.client.ui5.useIframe = true;
}
```